migrations/Version20240626130346.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20240626130346 extends AbstractMigration
```

Comment:
Do we need 2 separate migration files in this PR? Can't it be 1?

Reply:
done